Guided Hunting - Anomalous Office365 Exchange Sessions.ipynb

```diff
@@ -0,0 +1,547 @@
+{
```

Comment:
<p>Maybe combine these three cells so that you get some output when you run.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>